lib/posix/shell/Kconfig.uname

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) 2024 Meta
+# SPDX-License-Identifier: Apache-2.0
+
+if POSIX_UNAME
+
+config POSIX_UNAME_SHELL
+	bool "Support for `uname` command"
+	select SHELL_GETOPT
+	select POSIX_SHELL
```

Comment:
force pushed to make it such that the top level `posix` command is compiled only if there's a sub command